### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by mshahamc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,82 +1,52 @@
-# SampleCrmApp
```
Comment (ShadowRonin) [inline README.md]: Can you keep this section?

File: README.md [hunk 2]
```diff
@@ -1,82 +1,52 @@
...
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
-
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `npm run build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+#Deploying Sample CRM App locally:
```
Comment (ShadowRonin) [inline README.md]: This should be "Running Sample ..." not deploying

File: README.md (hunk 1 above)
Comment (mshahamc) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -1,15 +1,90 @@
...
+
+**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+
+If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+
+You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+
+## Learn More
```
Comment (ShadowRonin) [inline README.md]: Can you remove the Learn More section?

File: README.md [hunk 4]
```diff
@@ -1,15 +1,90 @@
...
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
+
+### `npm run build` fails to minify
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+
+
+
 # Running Sample CRM App locally:
```
Comment (ShadowRonin) [inline README.md]: Can you decrease the header size 1 level for this, creating the applications in studio, and integrating with salesforce? e.g. change # to ##

Comment (mshahamc) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (mshahamc) [reply README.md]: Fixed.

**Pull request merged**